Highrachy/ballers-api
# #172016511 Refactor linting in the codebase

#### What does this PR do?
This PR refactors linting in the codebase.

#### Description of Task to be completed?
Each of the tasks is done in a different commit for easy review:
* Fix Hound eslint parser error
* Configure Prettier to work with eslint (for visual studio code)
* Setup Husky for eslint and prettier

#### How should this be manually tested?
* Hound will not display the `parser error` any more
* On Visual studio code, installing the prettier plugin will no longer conflict with our eslint styles
* Husky will run when you try to commit a file. The staged files will be automatically prettified by prettier and eslint will try to fix any eslint errors (automatically) or print them out so that they can be fixed by you.

#### Screenshots (Husky in Action)
<img width="625" alt="Screenshot 2020-03-27 at 3 55 29 PM" src="https://user-images.githubusercontent.com/26963369/77768914-7407a800-7043-11ea-8f28-ac67fce2b74f.png">
